tests/VampTests.vala

```diff
@@ -0,0 +1,17 @@
+namespace Vamp {
+    class Test {
+        public static int main (string[] args) {
+            GLib.Test.init (ref args);
```

Comment:
If we are going to adopt the elementary code style, then it is not necessary to fully qualify the GLib names.

Reply:
No, but they conflict with the current class name. I was counting on separating tests on different classes, and the conflict will not exist there 😅

Reply:
Ah right, I just realized that this is because you created a class called Test.

In which case... why create a class in the first place? You don't need to :thinking:

Reply:
Coding style consistency in order to be 100% OOP 😂

Anyway, totally subject to taste...

Reply:
Would it make sense to call it UnitTest just to avoid clashes?

Reply:
Can be done, not a problem!

Reply:
Hmmm GitHub doesn't show me new comments unless I reload the site, how cool.

I don't support the motion to use OOP where it's not necessary. Unless we turn this into a full framework like libgee has, I don't see the need for classes, it's a bit overkill.